boat-scaffold/src/main/java/com/backbase/oss/codegen/angular/BoatAngularGenerator.java

```diff
@@ -100,7 +100,6 @@ public BoatAngularGenerator() {
             false));
         this.cliOptions.add(new CliOption(NG_VERSION, "The version of Angular. (At least 10.0.0)").defaultValue(this.ngVersion));
         this.cliOptions.add(new CliOption(FOUNDATION_VERSION, "The version of foundation-ang library.").defaultValue(this.foundationVersion));
-        this.cliOptions.add(new CliOption(SPEC_VERSION, "The version of OpenAPI YAML spec used to generate the NPM package.").defaultValue(this.specVersion));
```

Comment:
@ngohungphuc @santam85 this cli option should be included right? So devs know what the property is used for.